docs/examples/ampache_podcasts.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=ampache_podcasts
+After=network.target remote-fs.target nss-lookup.target
+Documentation=https://github.com/ampache/ampache/wiki
+
+[Service] 
+PrivateTmp=true 
+KillMode=mixed
+Type=oneshot
+User=www-data
+Group=www-data
+ExecStart= php bin/cli run:updateCatalog podcasts -a -g -i
+WorkingDirectory=/var/www/ampache
```

Comment:
Maybe this should match the WorkingDir in ampache_cron.service

Reply:
You are correct. I took it from my own setup